### ISSUE TITLE
Encode hash as Base64 in certificate names

### ISSUE THREAD

**Issue by atoms118** [atoms118/libpolynet-c#21]
This is dependent on #20, so for now at least it's parked

This would involve `pn_certificate_name_to_string` and `pn_string_to_certificate_name` in `certificate.c`. I believe there are already TODOs on lines `20` and `38`

**Comment by atoms118** [comment]
Ok so realistically this can get done before #20, it just can't get *tested* before #20
